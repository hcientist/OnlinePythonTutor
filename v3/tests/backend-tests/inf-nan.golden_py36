{
  "code": "x = float('inf')\ny = float('-inf')\nz = float('nan')\n",
  "trace": [
    {
      "line": 1,
      "event": "step_line",
      "func_name": "<module>",
      "globals": {},
      "ordered_globals": [],
      "stack_to_render": [],
      "heap": {},
      "stdout": ""
    },
    {
      "line": 2,
      "event": "step_line",
      "func_name": "<module>",
      "globals": {
        "x": [
          "SPECIAL_FLOAT",
          "Infinity"
        ]
      },
      "ordered_globals": [
        "x"
      ],
      "stack_to_render": [],
      "heap": {},
      "stdout": ""
    },
    {
      "line": 3,
      "event": "step_line",
      "func_name": "<module>",
      "globals": {
        "x": [
          "SPECIAL_FLOAT",
          "Infinity"
        ],
        "y": [
          "SPECIAL_FLOAT",
          "-Infinity"
        ]
      },
      "ordered_globals": [
        "x",
        "y"
      ],
      "stack_to_render": [],
      "heap": {},
      "stdout": ""
    },
    {
      "line": 3,
      "event": "return",
      "func_name": "<module>",
      "globals": {
        "x": [
          "SPECIAL_FLOAT",
          "Infinity"
        ],
        "y": [
          "SPECIAL_FLOAT",
          "-Infinity"
        ],
        "z": [
          "SPECIAL_FLOAT",
          "NaN"
        ]
      },
      "ordered_globals": [
        "x",
        "y",
        "z"
      ],
      "stack_to_render": [],
      "heap": {},
      "stdout": ""
    }
  ]
}
