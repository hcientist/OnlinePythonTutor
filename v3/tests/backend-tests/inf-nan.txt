x = float('inf')
y = float('-inf')
z = float('nan')
