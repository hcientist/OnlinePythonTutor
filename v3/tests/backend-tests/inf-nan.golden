{
  "code": "x = float('inf')\ny = float('-inf')\nz = float('nan')\n", 
  "trace": [
    {
      "ordered_globals": [], 
      "stdout": "", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {}, 
      "heap": {}, 
      "line": 1, 
      "event": "step_line"
    }, 
    {
      "ordered_globals": [
        "x"
      ], 
      "stdout": "", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {
        "x": [
          "SPECIAL_FLOAT", 
          "Infinity"
        ]
      }, 
      "heap": {}, 
      "line": 2, 
      "event": "step_line"
    }, 
    {
      "ordered_globals": [
        "x", 
        "y"
      ], 
      "stdout": "", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {
        "y": [
          "SPECIAL_FLOAT", 
          "-Infinity"
        ], 
        "x": [
          "SPECIAL_FLOAT", 
          "Infinity"
        ]
      }, 
      "heap": {}, 
      "line": 3, 
      "event": "step_line"
    }, 
    {
      "ordered_globals": [
        "x", 
        "y", 
        "z"
      ], 
      "stdout": "", 
      "func_name": "<module>", 
      "stack_to_render": [], 
      "globals": {
        "y": [
          "SPECIAL_FLOAT", 
          "-Infinity"
        ], 
        "x": [
          "SPECIAL_FLOAT", 
          "Infinity"
        ], 
        "z": [
          "SPECIAL_FLOAT", 
          "NaN"
        ]
      }, 
      "heap": {}, 
      "line": 3, 
      "event": "return"
    }
  ]
}
